### ISSUE TITLE
canvas.toBuffer('image/jpeg') returns undefined

### ISSUE THREAD

**Issue by donmccurdy** [Automattic/node-canvas#1923]
## Issue or Feature

When calling `canvas.toBuffer(mimeType)`, I'm getting a Buffer successfully for `image/png` but not `image/jpeg`. The second returns undefined. The code below worked on another machine, and on my test environment that runs node v12, v14, and v16, but is failing on my development machine. I'm wondering if this may be related to macOS Monterey 12.0.1 or the M1 Pro chip.

## Steps to Reproduce

```js
const {createCanvas} = require('canvas');

canvas = createCanvas(16, 32);
ctx = canvas.getContext('2d');
ctx.fillStyle = '#222222';

console.log('PNG: ' + !!canvas.toBuffer('image/png'));   // → true
console.log('JPEG: ' + !!canvas.toBuffer('image/jpeg')); // → false
```

## Your Environment
* Canvas: 2.8.0
* Environment: Node.js v14.18.1
* OS: macOS Monterey 12.0.1, M1 Pro

**Comment by donmccurdy** [comment]
Other dependencies in case they're relevant:

```shell
➜  glTF-Transform git:(main) ✗ brew install pkg-config cairo pango libpng jpeg giflib librsvg
Updating Homebrew...
==> Homebrew is run entirely by unpaid volunteers. Please consider donating:
  https://github.com/Homebrew/brew#donations
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/core).
==> New Formulae
cri-tools                            pdf2djvu                             sigrok-cli                           west                                 youtubeuploader
==> Updated Formulae
Updated 251 formulae.

Warning: pkg-config 0.29.2_3 is already installed and up-to-date.
To reinstall 0.29.2_3, run:
  brew reinstall pkg-config
Warning: cairo 1.16.0_5 is already installed and up-to-date.
To reinstall 1.16.0_5, run:
  brew reinstall cairo
Warning: pango 1.48.10 is already installed and up-to-date.
To reinstall 1.48.10, run:
  brew reinstall pango
Warning: libpng 1.6.37 is already installed and up-to-date.
To reinstall 1.6.37, run:
  brew reinstall libpng
Warning: jpeg 9d is already installed and up-to-date.
To reinstall 9d, run:
  brew reinstall jpeg
Warning: giflib 5.2.1 is already installed and up-to-date.
To reinstall 5.2.1, run:
  brew reinstall giflib
Warning: librsvg 2.50.7 is already installed and up-to-date.
To reinstall 2.50.7, run:
  brew reinstall librsvg
```

**Comment by zbjornson** [comment]
This might be fixed by #1910. Any chance you could try that PR out to help test it?

**Comment by donmccurdy** [comment]
@zbjornson thanks! Trying `npm install Automattic/node-canvas#m1` fixes the issue for me, yes. 👍

**Comment by zbjornson** [comment]
Was fixed in v2.9.0!
